vulnfeeds/cves/versions.go

```diff
@@ -89,6 +89,26 @@ func Repo(u string) (string, error) {
 			parsedURL.Hostname(), repo), nil
 	}
 
+	// cgit.freedesktop.org is a special snowflake with enough repos to warrant special handling
+	// it is a mirror of gitlab.freedesktop.org
```

Comment:
Can you add some examples in comments to demonstrate the variations of cgit.freedesktop.org URLs you're handling here?

Reply:
Done in 646884ca7bb9b91ff1ca1fdce1780c189e080f0d